### PR TITLE
Ds client

### PR DESCRIPTION
**Pull request by mdb**

### CODE REVIEW

File: traffic_ops/client/delivery_service.go [hunk 1]
```diff
@@ -128,6 +128,24 @@ type DeliveryServiceCapacity struct {
...
+
+// DeliveryServiceRouting ...
+type DeliveryServiceRouting struct {
+	StaticRoute       int     `json:"staticRoute"`
+	Miss              int     `json:"miss"`
+	Geo               float64 `json:"geo"`
+	Err               int     `json:"err"`
+	CZ                float64 `json:"cz"`
+	DSR               float64 `json:"dsr"`
+	FED               int     `json:"fed"`
```
Comment (mitchell852) [inline traffic_ops/client/delivery_service.go]: probably want to call this just Fed as it is not an acronym but rather short for Federation

File: traffic_ops/client/fixtures/delivery_service.go [hunk 2]
```diff
@@ -108,3 +108,19 @@ func DeliveryServiceCapacity() *client.DeliveryServiceCapacityResponse {
...
+
+// DeliveryServiceRouting returns a default DeliveryServiceRoutingResponse to be used for testing.
+func DeliveryServiceRouting() *client.DeliveryServiceRoutingResponse {
+	dsc := client.DeliveryServiceRouting{
+		StaticRoute: 1,
+		Miss:        2,
+		Geo:         3.33,
+		Err:         4,
+		CZ:          5.55,
+		DSR:         6.66,
```
Comment (mitchell852) [inline traffic_ops/client/fixtures/delivery_service.go]: did you want to add Fed, RegionalAlternate, and RegionalDenied to this fixture?

File: docs/source/development/traffic_ops_api/v12/deliveryservice.rst [hunk 3]
```diff
@@ -543,22 +543,31 @@ Health
...
+  +--------------------------+--------+-----------------------------------------------------------------------------------------------------------------------------+
+  | ``cz``                   | number | The percentage of Traffic Router requests for this deliveryservice satisfied by a CZF hit.                                  |
+  +--------------------------+--------+-----------------------------------------------------------------------------------------------------------------------------+
+  | ``dsr``                  | number | The percentage of Traffic Router requests for this deliveryservice satisfied by sending the                                 |
+  |                          |        | client to the overflow CDN.                                                                                                 |
+  +--------------------------+--------+-----------------------------------------------------------------------------------------------------------------------------+
+  | ``fed``                  | number |                                                                                                                             |
```
Comment (mitchell852) [inline docs/source/development/traffic_ops_api/v12/deliveryservice.rst]: i will submit a PR to your branch for these missing descriptions

File: traffic_ops/client/delivery_service.go (hunk 1 above)
Comment (mdb) [inline traffic_ops/client/delivery_service.go]: Ah, thank you; that makes sense. Not sure why I assumed it to be an acronym.

**Comment by mdb** [comment]
@mitchell852 I believe I've made the requested changes; please let me know if I'm mistaken.

**Pull request merged**